src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy

```diff
@@ -442,6 +442,31 @@ class GitVersionPluginTests extends Specification {
         buildResult.output =~ ":printVersionDetails\n1.0.0\n"
     }
 
+    def 'git describe with commit after tag' () {
+        given:
+        buildFile << '''
+            plugins {
+                id 'com.palantir.git-version'
+            }
+            version gitVersion()
+        '''.stripIndent()
+        gitIgnoreFile << 'build'
+        Git git = Git.init().setDirectory(projectDir).call();
+        git.add().addFilepattern('.').call()
+        git.commit().setMessage('initial commit').call()
+        git.tag().setAnnotated(true).setMessage('1.0.0').setName('1.0.0').call()
+        dirtyContentFile << 'dirty-content'
+        git.add().addFilepattern('.').call()
+        git.commit().setMessage('added some stuff').call()
+
+        when:
+        BuildResult buildResult = with('printVersion').build()
+        String commitSha = (++git.log().call().iterator()).getName()
```

Comment:
I haven't seen this '++' syntax used this way before, is there any way to get the most recent commit SHA out of the `git.commit()` invocation above instead?